backend/game/tests.py

```
@@ -1883,3 +1884,100 @@ def test_the_campus_socket_has_a_route(self):
+        # Stored happily, it would fail to load for everyone who met this
+        # player — a broken avatar for the whole room rather than an error for
+        # the one who asked for it.
+        response = self.client.patch(
+            '/api/auth/user/', {'campus_character': 'wizard'}, format='json'
+        )
+        self.assertEqual(response.status_code, 400)
+        self.user.refresh_from_db()
+        self.assertEqual(self.user.campus_character, '')
+
+    def test_nobody_can_dress_anybody_else(self):
+        other = User.objects.create_user(username='someone-else', password='pw')
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to argument: "password"

(S106)